tests/unit/backend/corpora/common/entities/datasets/test_revision.py

```diff
@@ -19,7 +19,9 @@ def test__create_dataset_revision(self):
         with self.subTest("deployment is correctly created and points to correct s3 uri "):
             rev_deployment = rev_dataset.pop("explorer_url")
             original_deployment = dataset.pop("explorer_url")
-            self.assertEqual(rev_deployment, original_deployment)
+            self.assertIsNotNone(original_deployment)
+            # explorer_url isn't set until a new dataset is uploaded
+            self.assertIsNone(rev_deployment)
```

Comment:
So `rev_deployment` being None is the correct test for "deployment is correctly created and points to correct s3 uri"?

Reply:
prior to uploading a dataset the explorer_url should be set to None to avoid issues identifying a dataset based on its explorer_url. Once a dataset is uploaded for the revision an explorer_url will be created as part of the upload process